classes/submission/Collector.inc.php

```diff
@@ -93,13 +93,15 @@ public function getQueryBuilder(): Builder
                 foreach ($assocIds as $assocId) {
                     $join->on('sf.assoc_id', '=', DB::raw(intval($assocId)));
                 }
+
+                $join->groupBy(DB::raw('sf.seq'));
```

Comment:
The `index.php/publicknowledge/catalog` was getting a 500 error due to the `groupBy`. Moved it to `leftJoin` statement and the page works again.

Reply:
Hmm, I think I'd like a little more detail on this before adding a `groupBy`... What was the 500 error you were getting? Where is it moved to `leftJoin`? Is this on your own machine, or in Travis testing, or both?

Reply:
It's failing on both.

Here's the Travis job https://app.travis-ci.com/github/pkp/omp/jobs/549610608

And I just move the groupBy from line 100, to inside the leftJoin, thus works.

Reply:
I can reproduce this locally by turning on `ONLY_FULL_GROUP_BY` in MySQL:
```
Syntax error or access violation: 1055 'omp-main.s.submission_id' isn't in GROUP BY (SQL: select `s`.`submission_id` from `submissions` as `s` left join `publications` as `po` on `s`.`current_publication_id` = `po`.`publication_id` left join `features` as `sf` on `s`.`submission_id` = `sf`.`submission_id` and `sf`.`assoc_type` = 512 and `sf`.`assoc_id` = 1 where `s`.`context_id` in (1) and `s`.`status` in (3) group by sf.seq order by case when sf.seq is null then 1 else 0 end, `sf`.`seq` ASC, `po`.`date_published` desc)
```
The best solution is to [avoid `groupBy` clauses entirely](https://github.com/pkp/pkp-lib/issues/7353), unless they're strictly necessary. I've merged this upstream:
https://github.com/pkp/omp/commit/b31d597fe49ab221c564d0d991f87502578664de

It works fine on my MariaDB with `ONLY_FULL_GROUP_BY` enabled, so should pass Travis.

Reply:
Thank you @asmecher